r/DESCRIPTION

```diff
@@ -43,7 +43,7 @@ Imports:
     utils,
     vctrs
 Roxygen: list(markdown = TRUE, r6 = FALSE, load = "source")
-RoxygenNote: 7.2.3
+RoxygenNote: 7.3.1
```

Comment:
I'm happy to back this out + the extra docs changes, but they were stale on main.